setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 NAME = "crossengage-client"
-VERSION = "1.1.0"
+VERSION = "1.2.0"
```

Comment:
Maybe we shouldn't have a `1.*` version for an as yet unused service. 😉 

What about `0.2.0` instead?